packages/jitm/src/JITM.php

```diff
@@ -149,6 +153,116 @@ static function jitm_jetpack_woo_services_activate( $CTA ) {
+		_e( '<h2>Important notice about your Jetpack connection: </h2>', 'jetpack' );
+		printf(
+			__( "<p>Warning! You are about to delete the Jetpack <a href='%1\$1s' target='_blank'>connection owner</a> (%2\$2s) for this site, which may affect some of your features.</p>", 'jetpack' ),
+			'https://jetpack.com/support/primary-user/',
+			esc_html( $connection_owner->data->user_login )
+		);
+
+		if ( ! empty( $connected_users ) && count( $connected_users ) > 1 ) {
+			echo '<form id="jp-switch-connection-owner" action="" method="post">';
+			echo "<label for='owner'>" . __( 'You can choose to transfer connection ownership to one of these already-connected admins: ', 'jetpack' ) . '</label>';
+
+			$connected_user_ids = array_map(
+				function( $connected_user ) {
+						return $connected_user->ID;
+				},
+				$connected_users
+			);
+
+			wp_dropdown_users(
+				array(
+					'name'    => 'owner',
+					'include' => array_diff( $connected_user_ids, array( $connection_owner->ID ) ),
+					'show'    => 'display_name_with_login',
+				)
+			);
+
+			submit_button( __( 'Set new connection owner', 'jetpack' ), 'primary', 'jp-switch-connection-owner-submit' );
+
+			_e( '<p>As always, feel free to <a href="https://jetpack.com/contact-support" target="_blank">contact our support team</a> if you have any questions.</p>', 'jetpack' );
+
+			echo "<div id='jp-switch-user-results'></div>";
+			echo '</form>';
+			?>
+			<script type="text/javascript">
+				jQuery( document ).ready( function( $ ) {
+					$( '#jp-switch-connection-owner' ).on( 'submit', function( e ) {
+						const formData = $( this ).serialize();
+						const submitBtn = document.getElementById( 'jp-switch-connection-owner-submit' );
+						const results = document.getElementById( 'jp-switch-user-results' );
+
+						submitBtn.disabled = true;
+
+						$.ajax( {
+							type        : "POST",
+							url         : "<?php echo get_rest_url() . 'jetpack/v4/connection/owner'; ?>",
+							data        : formData,
+							headers     : {
+								'X-WP-Nonce': "<?php echo wp_create_nonce( 'wp_rest' ); ?>",
+							},
+							success: function() {
+								results.innerHTML = "Success! thank you.";
+								setTimeout( function() {
+									$( '#jetpack-notice-switch-connection-owner' ).hide( 'slow' );
+								}, 1000 );
+							}
+						} ).done( function() {
+							submitBtn.disabled = false;
+						} );
+
+						e.preventDefault();
+						return false;
+					} );
+				} );
+			</script>
+			<?php
+		} else {
+			_e( '<p>Unfortunately, there are no other connected admins to transfer the connection to.</p>', 'jetpack' );
+			printf(
+				__( '<p>If you would like to be the new connection owner for this site, please connect to your WordPress.com account by clicking <a href="%s" target="_blank">this link</a>. Once you connect, you may refresh this page to see an option to change the connection owner.</p>', 'jetpack' ),
```

Comment:
What do you think if you'd move the link from "this link" to "connect..."?

> If you would like to be the new connection owner for this site, please connect to your WordPress.com account by clicking [this link].
→
> If you would like to be the new connection owner for this site, please [connect to your WordPress.com account].